tests/e2e/helpers/conf.ts

```diff
@@ -19,6 +19,14 @@ export const ossStandaloneConfig = {
     databasePassword: process.env.OSS_STANDALONE_PASSWORD
 };
 
+export const ossStandaloneConfig2 = {
+    host: process.env.OSS_STANDALONE_HOST || 'oss-standalone-2',
+    port: process.env.OSS_STANDALONE_PORT || '6379',
+    databaseName: `${process.env.OSS_STANDALONE_DATABASE_NAME || 'test_standalone'}-${uniqueId}`,
```

Comment:
the same name as for ossStandaloneConfig
please rename to 'test_standalone_empty'}-${uniqueId}`